bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/dialogs/YesNoCancelListSelectionDialog.java

```diff
@@ -27,7 +27,7 @@
  *             to subclass the regular ListSelectionDialog, which uses
  *             OK/Cancel, and provide a separate checkbox if necessary.
  */
-@Deprecated
+@Deprecated(forRemoval = true, since = "2024.03")
```

Comment:
The format for the since is 2024-03

Reply:
Done

Reply:
Would you please make it 2025-03 :) ? A year late we dhould be able to complete it .